README.md

```diff
@@ -58,29 +58,29 @@ While other versions often work, the above are what we officially support. YMMV!
 
 | High-Level Packages                                            | Responsibility                                               |
 | -------------------------------------------------------------- | ------------------------------------------------------------ |
 | [@taquito/taquito](packages/taquito)                           | [Facade](https://en.wikipedia.org/wiki/Facade_pattern) to lower-level, package-specific functionality  |
 
 | Low-Level Packages                                               | Responsibility                                                |
 | ---------------------------------------------------------------- | ------------------------------------------------------------- |
-| [@taquito/local-forging](packages/taquito-local-forging)         | Local "forging" of Tezos operations                           |
-| [@taquito/michelson-encoder](packages/taquito-michelson-encoder) | Creates a JS abstraction for Smart Contracts                  |
-| [@taquito/michel-codec](packages/taquito-michel-codec)           | Converts Michelson between forms and expands Macros           |
+| [@taquito/local-forging](packages/taquito-local-forging)         | Local "forging" (preparation) of Tezos operations             |
+| [@taquito/michelson-encoder](packages/taquito-michelson-encoder) | Creates JS abstractions of Smart Contracts                    |
+| [@taquito/michel-codec](packages/taquito-michel-codec)           | Converts Michelson between forms, expands Macros, etc         |
 | [@taquito/remote-signer](packages/taquito-remote-signer)         | Provides the facility to use a remote signer, such as https://signatory.io    |
 | [@taquito/rpc](packages/taquito-rpc)                             | RPC client library: every rpc endpoint has its own method     |
-| [@taquito/signer](packages/taquito-signer)                       | Provide necessary function to sign using tezos keys           |
-| [@taquito/utils](packages/taquito-utils)                         | Provide different encoding and decoding utilities             |
-| [@taquito/tzip12](packages/taquito-tzip12)                       | TZIP-12 Implementation for Taquito                            |
-| [@taquito/tzip16](packages/taquito-tzip16)                       | TZIP-16 Implementation for Taquito                            |
-| [@taquito/beacon-wallet](packages/taquito-beacon-wallet)         | TZIP-10 Wallet Interaction implementation for the Wallet API  |
+| [@taquito/signer](packages/taquito-signer)                       | Provides functionality to sign data using tezos keys          |
+| [@taquito/utils](packages/taquito-utils)                         | Provides different encoding and decoding utilities            |
+| [@taquito/tzip12](packages/taquito-tzip12)                       | TZIP-12 implementation for Taquito, related to NFTs           |
```

Comment:
I wonder if "related to NFTs" is too vague. The current implementation of tzip 12 and 1zip16 only relate to metadata.

tzip 12 allows retrieving token metadata
tzip 16 allows retrieving contract metadata and executing off-chain views.

Reply:
Thank you - I was hand-waving and this is far better.